articles/openstack/ostack-gs.md

```diff
@@ -497,7 +501,7 @@ Whatever library or automation tools you choose to use, you'll need to know a fe
 
 ## Using the OpenStack command-line client
 
-The OpenStack project provides a command‑line client that enables you to access the project API through easy‑to‑use commands. For example, the Compute service provides a NOVA command‑line client.
+The OpenStack project provides a command‑line client that enables you to access the project API through easy‑to‑use commands. For example, the compute service provides a NOVA command‑line client.
 
 You can run the commands from the command line, or include the commands in scripts to automate tasks. If you provide OpenStack credentials, such as your user name and password, you can run these commands on any computer.
```

Comment:
How does the login process work for the OpenStack CLI with SSO/2FA? For OpenShift, we have the following:

If your OpenShift cluster uses OpenShift Container Platform (OCP) 3.10 or later, when you enter the login command, you'll be given a URL to use to log in using your Portal credentials. Go to this URL and you'll be prompted for your Portal credentials and, if you've set up 2FA, your 2FA code. This will take you to a page with your API token, the command to run to log in to the CLI and curl request to use to log in to the CLI.

Reply:
API RC changes added.